### ISSUE TITLE
Create doc for installing on raw k8s without Helm/Tiller

### ISSUE THREAD

**Issue by tilleryd** [DecipherNow/helm-charts#353]
Recently, we provided raw k8s YAML generated from a dry-run of our Helm charts. This is a quick way to get a customer up and running on k8s if they can't use Helm/Tiller. @kaitmore has a nice templated solution on top of this to apply values similar to what we are doing with Helm.

Let's get this documented as a fallback to Helm in this repo.

**Comment by chrisbsmith** [comment]
While I agree that this is a necessary document to create, I would argue that it doesn't belong in the `helm-charts` repo.  Alas, I'm truly indifference to where it ends up, but figure it should probably be somewhere else.

**Comment by kaitmore** [comment]
@chrisbsmith @tilleryd I agree with Chris, perhaps it can go in the greymatter repo? It's a little funny to have a doc for "deploying without helm" in the helm charts repo 😆

**Comment by tilleryd** [comment]
That’s fine

**Comment by kaitmore** [comment]
@tilleryd @chrisbsmith 

After actually writing instructions out, I'm starting to think Helm charts _is_ the right place to put this doc after all, since this is really about using Helm as a templating engine to generate configs. The steps are essentially just 1. fill out greymatter.yaml with your deployment config and 2. run helm template with the filled out custom files.

For DHS, we didn't know a lot of the deployment config ahead of time, so I filled out greymatter.yaml with a bunch of placeholder variables (e.g. `__NAMESPACE__`) and ran helm template, then I created a lookup table for DHS with notes on how to fill things in. I was initially thinking of reusing the greymatter.yaml with placeholders, but then I would have to keep that in sync with the normal greymatter.yaml.

I've created this PR here https://github.com/DecipherNow/helm-charts/pull/358 If you guys disagree after reading it, I'll go ahead and move the PR to a more appropriate repo. LMK what you think.
